R/rdeephaven/R/update_by_ops_wrapper.R

```diff
@@ -361,7 +373,7 @@ udb_roll_group_time <- function(ts_col, cols, rev_time, fwd_time = "PT0s") {
 #' @param rev_ticks Integer scalar denoting the look-behind window size in number of rows.
 #' @param fwd_ticks Integer scalar denoting the look-ahead window size in number of rows. Default is 0.
 #' @export
-udb_roll_avg_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
+uby_rolling_avg_tick <- function(cols, rev_ticks, fwd_ticks = 0) {
```

Comment:
If this is a simple moving average, that and SMA should be indicated in the docstring.